composer.json

```diff
@@ -51,5 +52,10 @@
         "platform": {
             "php": "7.0"
         }
+    },
+    "scripts": {
+        "phpstan": [
+            "./vendor/bin/phpstan analyse --level 5 src"
```

Comment:
@holtkamp Why do we go exactly for `level 5`?

Reply:
no particular reason... as a bank you guys should probably "strive" for level 7 😄 
https://github.com/phpstan/phpstan#rule-levels

Reply:
@holtkamp what is the highest level the SDK is passing at the moment?